kernel/src/engine/default/filesystem.rs

```diff
@@ -216,6 +218,33 @@ mod tests {
         assert_eq!(data[2], Bytes::from("el-da"));
     }
 
+    #[tokio::test]
+    async fn test_file_meta_is_correct() {
+        let store = Arc::new(InMemory::new());
+
+        let begin_time = SystemTime::now().duration_since(UNIX_EPOCH).unwrap();
+        // The [`FileMeta`]s must be greater than 1 minute ago
+        let allowed_time = begin_time - Duration::from_secs(60);
```

Comment:
qq: Why do we need such a big safety margin, out of curiosity? All files are PUT after this timestamp, so they should not have smaller timestamps?

Reply:
Could we also just assert the file_meta time is < current time (and could include a little safety margin as well)? Right now we only have a bound on one end (I don't think this would catch the case of us accidentally returning nanos instead of millis for example)